src/test/java/com/snowflake/kafka/connector/internal/JdbcPropertiesTest.java

```diff
@@ -0,0 +1,73 @@
+package com.snowflake.kafka.connector.internal;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.junit.Assert.*;
+
+import java.util.Properties;
+import org.junit.Test;
+
+public class JdbcPropertiesTest {
+
+  @Test
```

Comment:
Please add new tests using JUnit5 (import from `junit.jupiter` package)

Reply:
Done in both tests.

Reply:
Done in both classes.

Reply:
Thanks!